### DISCUSSION TITLE
Slack link seems to have expired in README

### DISCUSSION THREAD

**Discussion by jasonribble** [documenso/documenso#198] [General]
Howdy! 

I'm very excited about documenso. This will allow business from around the world. I'd love to chat with y'all. It seems that the link provided in the Github is no longer availabble.

**Comment by ephraimduncan** [comment]
You can join with this link. 

https://join.slack.com/t/documenso/shared_invite/zt-1wrwmi601-Zcn1GpVOagcakE842fVCBQ

**Comment by ElTimuro** [comment]
closing as this seems resolved :)

**Comment by ElTimuro** [comment]
Also: Thanks for the kind words @jasonribble
